README.md

```diff
@@ -124,3 +124,17 @@ These primitives are only defined when the result itself is a pure rational.
 
 Use double quotes to get a quote: `"Foo "" Bar"` is `Foo " Bar`.
 No primitives defined.
+
+## Benchmarks
+
+We run all of our "examples" [as benchmarks in codspeed](https://codspeed.io/egraphs-good/egglog). These are in CI
+for every commit in main and for all PRs. It will run the examples with extra instrumentation added so that it can
+capture a single trace of the CPU interactions ([src](https://docs.codspeed.io/features/understanding-the-metrics/)):
+
+> CodSpeed instruments your benchmarks to measure the performance of your code. A benchmark will be run only once and the CPU behavior will be simulated. This ensures that the measurement is as accurate as possible, taking into account not only the instructions executed but also the cache and memory access patterns. The simulation gives us an equivalent of the CPU cycles that includes cache and memory access.
+
+Since many of the shorter running benchmarks have unstable timings due to non deterministic behavior, we ["ignore"](https://docs.codspeed.io/features/ignoring-benchmarks/)
+them in codspeed. That way, we still capture their performance, but their timings don't show up in our reports by default.
+
+We use 50ms as our cutoff currently, any benchmarks shorter than that are ignored. This number was selected to try to ignore
```

Comment:
Is this automatic or manual? This description makes it sound automatic but the CodSpeed docs above make it sound manual. (If it's automatic no changes are necessary.)

Reply:
Manual, I added a clarification that's the case